### PR TITLE
Create Dockerhub GitHub action

### PR DESCRIPTION
**Pull request by Andrew-Chen-Wang**
I saw a few comments about just consistently updating per master push, so I set up the GitHub action to build on every commit. Hope this helps.

You'll just need a Docker repository and your username and password in GitHub secrets.

### CODE REVIEW

File: .github/workflows/dockerhub.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+        - master
+
+jobs:
+  build:
+    name: Build NGINX Image
```
Comment (mxrch) [inline .github/workflows/dockerhub.yml:11]: Ah I think you forgot changing this line 😄

**Review by mxrch — CHANGES_REQUESTED** [review]

File: .github/workflows/dockerhub.yml (hunk 1 above)
Comment (Andrew-Chen-Wang) [reply .github/workflows/dockerhub.yml:11]: Lol I copied and pasted from my own repo.

**Comment by mxrch** [comment]
Alright I'm doing this tonight or tomorrow, thanks for the PR !

**Comment by mxrch** [comment]
@Andrew-Chen-Wang can I contact you on Discord to help me setup the Github action ? now the build is working but I see nothing on Dockerhub :'(

**Comment by Andrew-Chen-Wang** [comment]
Yea I can join the server

**Pull request merged**